lib/nfg_ui/components/traits/typeface.rb

```diff
@@ -10,7 +10,6 @@ module Typeface
                     heading
                     subheading
                     title
-                    muted
```

Comment:
@farhajawed typeface no longer accepts muted trait, in case this was documented. probably wasn't.

Reply:
I am a bit confused with this.
Should the following with `:muted` trait work 
`= ui.nfg :typeface, :muted, caption: 'Used for block buttons spanning the width of container.', class: 'mb-0'`?
Looks like it works.

Reply:
Hey @farhajawed thanks for spotting. Looks like this was more of a cleanup issue -- `Typeface` component includes a special Trait module called `NfgUi::Components::Traits::Muted`  -- this must be how muted is still being applied and was able to be removed from the typeface traits. We should be good to go.